### DISCUSSION TITLE
Become A Members

### DISCUSSION THREAD

**Discussion by tcf-admin** [tcfev/task-force-nika#2] [General]
You can become a member of task-force-nika in two ways, anonymously and legally.
 
Anonymously:
1. Create an anonymous Github account (if you have security concerns - we recommend it) 
2. Announce your willingness and expertise here in this discussion
3. Work directly on open-source projects, and check out our issues and [backlog](https://github.com/orgs/tcfev/projects/12)
4. Welcome aboard!

Legally:
1. Ask for tcf membership [here](https://transcf.org/en/membership)
2. One of our members will interview you
3. The [constitution](https://github.com/tcfev/task-force-nika/blob/main/assets/English-bylaws-ratified-on-Dec-2022.pdf) and [NDA](https://github.com/tcfev/task-force-nika/blob/main/assets/privay-policy-data-protection-en.pdf) will be sent to you to sign digitally
4. Welcome aboard!

**Comment by JJScubaKing** [comment]
Hi,

I am JJ, with 5 years of experience in lean startup development and product design. I would love to be able to help with your ongoing and future projects as well as share my own with your team. 

Thanks in advance, 
JJ

**Comment by Jiwe-Mobarez** [comment]
Hi, I am a web developer currently living in Iran. I enjoy designing systems and writing about ideas.

I think I can help with the public facing side of the organization quite a lot. If there are Ideas that need to be advertised, or argued for, I'd love to help.

**Comment by Decentralized-Singularity** [comment]
Hi there, 
I'm a researcher in the environmental engineering field and I'm interested in system resilience. 
'TrueWE' is an idea that we want to develop and implement together which increases life resilience within a decentralized pattern based on permaculture principles.   
I would appreciate your insights and support further into the project. 

Cheers,
TrueWE.

**Comment by ghost** [comment]
Hello. I'm a software engineer working on a few projects similar to what others are doing. I would like to join so we can coordinate and combine our strengths.
